pyanaconda/modules/timezone/timezone.py

```diff
@@ -210,5 +211,8 @@ def install_with_tasks(self):
                 sysroot=conf.target.system_root,
                 ntp_enabled=self.ntp_enabled,
                 ntp_servers=self.time_sources
+            ),
+            ConfigureHwClockTask(
```

Comment:
Could you move the task on the top of the list, please? We are already changing its position, so let's keep at least the order of these tasks.